pythonforandroid/build.py

```diff
@@ -621,10 +622,8 @@ def run_setuppy_install(ctx, project_dir, env=None):
             ),
             "freeze"
         ], env=copy.copy(env))
-        try:
+        with suppress(AttributeError):
             constraints = constraints.decode("utf-8", "replace")
-        except AttributeError:
-            pass
```

Comment:
:fire: two less  lines of code  without sacrificing readability :smile:

Reply:
Yes I love that one :)
But it cannot really be used nicely if we have a large comment block above the `pass`, so yes I decided to introduced it only when we had a simple except ...: pass